components/ChoreWeight.tsx

```diff
@@ -1,6 +1,7 @@
-import { useState } from 'react';
+import { useContext, useState } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 import { Badge, Card } from 'react-native-paper';
```

Comment:
Text is being imported from react-native when it should be imported from paper
```suggestion
import { StyleSheet, TouchableOpacity, View } from 'react-native';
import { Badge, Text, Card } from 'react-native-paper';
```